rasa/cli/data.py

```diff
@@ -121,28 +128,28 @@ def _add_data_convert_parsers(
         help="Migrate model configuration between Rasa Open Source versions.",
     )
     migrate_config_parser.set_defaults(func=_migrate_model_config)
-    _add_migrate_config_arguments(migrate_config_parser)
-
-
-def _add_migrate_config_arguments(parser: argparse.ArgumentParser) -> None:
```

Comment:
Not sure here - why do we remove `_add_migrate_config_arguments`?
IMO we can have a new `_add_convert_responses_arguments` instead.

Reply:
initially the method was here (made by @wochinge I think) because it did the legwork of adding custom arguments. I realised that it could leverage the `default_arguments` module, so I made the change in [this commit](https://github.com/RasaHQ/rasa/pull/7183/commits/9ef7bbea43468bda1dea79c7a65b0eb96b9d5c7f). The code is shorter and leverages existing interface. Since it's very short now, I thought that containing it in `_add_migrate_config_arguments()` became useless